<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>633</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>BluePY</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/img/LOGO.PNG</normaloff>:/resource/img/LOGO.PNG</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget{
	border-image: url(:/resource/img/bg_main.jpg);
    border: 2px solid #ffffff;
    border-radius: 20px;
    min-width: 80px;
}

QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QWidget#page_chart{
	border-radius:10px;
	background-color: rgb(214, 214, 214);
}

QWidget#page_report{
	border-radius:10px;
	background-color: rgb(214, 214, 214);
}

QWidget#menu{
	border-radius : 10px;
	background-color: rgba(41, 105, 255, 80);
}

QPushButton#BT_Server{
border-image: url(:/resource/img/server.png);
}

QPushButton#BT_Search{
border-image: url(:/resource/img/search-1.png);
}

QPushButton#BT_Explorer{
border-image: url(:/resource/img/browser-1.png);
}

QPushButton#BT_Home{
border-image: url(:/resource/img/home.png);
}

QPushButton#BT_Save{
border-image: url(:/resource/img/disk.png);
}

QPushButton#BT_Exit{
border-image: url(:/resource/img/logout.png);
}

QPushButton#BT_Help{
border-image: url(:/resource/img/question.png);
}

QPushButton#BT_Info{
border-image: url(:/resource/img/customer-service.png);
}

QPushButton#BT_Config{
border-image: url(:/resource/img/config.png);
}

QTreeWidget#treeWidget{
    border: 2px solid #ffffff;
    border-radius : 10px;
}

QToolBar#mainToolBar{
background-color: rgb(236, 236, 236);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="menu" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="BT_Server">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Search">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Explorer">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Config">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Home">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Help">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Info">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BT_Exit">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
           </property>
           <column>
            <property name="text">
             <string>Server List</string>
            </property>
            <property name="font">
             <font>
              <family>Sitka Heading</family>
              <pointsize>20</pointsize>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="background">
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </property>
            <property name="foreground">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Linux</string>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/resource/img/linux.png</normaloff>:/resource/img/linux.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Window</string>
            </property>
            <property name="font">
             <font>
              <family>바탕</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/resource/img/windows.png</normaloff>:/resource/img/windows.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="palette">
            <palette>
             <active/>
             <inactive/>
             <disabled/>
            </palette>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="page_main"/>
           <widget class="QWidget" name="page_chart">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>HY수평선M</family>
                   <pointsize>25</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>점검 결과</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTableWidget" name="tableWidget">
               <column>
                <property name="text">
                 <string>CODE</string>
                </property>
                <property name="background">
                 <color>
                  <red>220</red>
                  <green>234</green>
                  <blue>245</blue>
                 </color>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>RISK</string>
                </property>
                <property name="background">
                 <color>
                  <red>220</red>
                  <green>234</green>
                  <blue>245</blue>
                 </color>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>RESULT</string>
                </property>
                <property name="background">
                 <color>
                  <red>220</red>
                  <green>234</green>
                  <blue>245</blue>
                 </color>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>EXPLAIN</string>
                </property>
                <property name="background">
                 <color>
                  <red>220</red>
                  <green>234</green>
                  <blue>245</blue>
                 </color>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_report">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>HY수평선M</family>
                   <pointsize>25</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>취약 항목 보고</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>89</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QScrollArea" name="scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>828</width>
                  <height>446</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="report_img">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionServer">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/server.png</normaloff>:/resource/img/server.png</iconset>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
   <property name="toolTip">
    <string>Server</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/search-1.png</normaloff>:/resource/img/search-1.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
  <action name="actionFinder">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/browser-1.png</normaloff>:/resource/img/browser-1.png</iconset>
   </property>
   <property name="text">
    <string>Finder</string>
   </property>
   <property name="toolTip">
    <string>Finder</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/disk.png</normaloff>:/resource/img/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/logout.png</normaloff>:/resource/img/logout.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/question.png</normaloff>:/resource/img/question.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/customer-service.png</normaloff>:/resource/img/customer-service.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Info</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/home.png</normaloff>:/resource/img/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/setting.PNG</normaloff>:/resource/img/setting.PNG</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="toolTip">
    <string>Setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
